build/package/rpm/go-fdo-server.spec

```diff
@@ -3,23 +3,26 @@
 
 # https://github.com/fido-device-onboard/go-fdo-server
 %global goipath         github.com/fido-device-onboard/go-fdo-server
+%global gourl           https://%{goipath}
 %global debug_package   %{nil}
 #%%global tag             v0.0.0
 #%%global commit          08a576eb601fcdb94ee7fa219c16c02834a649e5
+%global combined_license  Apache-2.0 AND BSD-3-Clause AND MIT
 
 Name:           go-fdo-server
 Version:        0
 Release:        %autorelease -p
 Summary:        A Go implementation of the FIDO Device Onboard Specification
 
 # Generated by go-vendor-tools
-License:        Apache-2.0 AND BSD-3-Clause AND MIT
+License:        %{combined_license}
 URL:            %{gourl}
 Source0:        %{gosource}
 # Generated by go-vendor-tools
 Source1:        %{name}-%{version}-vendor.tar.gz
 Source2:        go-vendor-tools.toml
 
+%gometa -L -f
```

Comment:
I had to move this *above* %description: there's a bug where part of the directives generated by %gometa appear in the %description otherwise!

```go-fdo-server.src: W: tag-in-description BuildRequires:
go-fdo-server.src: W: tag-in-description ExclusiveArch:
go-fdo-server.x86_64: W: tag-in-description BuildRequires:
go-fdo-server.x86_64: W: tag-in-description ExclusiveArch:
```